commands/get/resources/getter.go

```diff
@@ -0,0 +1,145 @@
+func NewVCSAndLocalGetter(parser opencontrol.SchemaParser) Getter {
+	return &vcsAndLocalFSGetter{Downloader: NewVCSDownloader(), FSUtil: fs.OSUtil{}, Parser: parser, ResourceMap: mapset.Init()}
+}
+
+// vcsAndLocalFSGetter is the resource getter that uses VCS for remote resource getting and local file system for local resources.
+type vcsAndLocalFSGetter struct{
+	Downloader Downloader
+	FSUtil fs.Util
+	ResourceMap mapset.MapSet
+	Parser      opencontrol.SchemaParser
+}
+
+// GetLocalResources is the implementation that uses the local file system to get local resources.
+func (g *vcsAndLocalFSGetter) GetLocalResources(source string, resources []string, destination string, subfolder string, recursively bool, resourceType constants.ResourceType) error {
+	for _, resource := range resources {
+		if result := g.ResourceMap.Reserve(string(resourceType), resource); !result.Success {
+			return result.Error
+		}
+		resourceSource := filepath.Join(source, resource)
+		resourceDestinationFolder := filepath.Join(destination, subfolder)
+		resourceDestination := filepath.Join(resourceDestinationFolder, filepath.Base(resource))
+		var err error
+		log.Printf("Ensuring directory %s exists\n", resourceDestinationFolder)
+		err = g.FSUtil.Mkdirs(resourceDestinationFolder)
+		if err != nil {
+			return err
+		}
+		log.Printf("Attempting to copy local resource %s into %s\n", resourceSource, resourceDestination)
+		if recursively {
+			log.Printf("Copying local resource %s reursively into %s\n", resourceSource, resourceDestination)
+			err = g.FSUtil.CopyAll(resourceSource, resourceDestination)
+		} else {
+			log.Printf("Copying local resource %s into %s\n", resourceSource, resourceDestination)
+			err = g.FSUtil.Copy(resourceSource, resourceDestination)
+		}
+		if err != nil {
+			log.Printf("Copying local resources %s failed\n", resourceSource)
+			return err
+		}
+	}
+	return nil
+}
+
+// GetRemoteResources is the implementation that uses VCS to get remote resources.
+func (g *vcsAndLocalFSGetter) GetRemoteResources(destination string, subfolder string, entries []common.RemoteSource) error {
+	tempResourcesDir, err := g.FSUtil.TempDir("", "opencontrol-resources")
+	if err != nil {
+		return err
+	}
+	defer os.RemoveAll(tempResourcesDir)
+	for _, entry := range entries {
+		tempPath := filepath.Join(tempResourcesDir, subfolder, filepath.Base(entry.GetURL()))
+		// Clone repo
+		log.Printf("Attempting to clone %v into %s\n", entry, tempPath)
+		err := g.Downloader.DownloadRepo(entry, tempPath)
+		if err != nil {
+			return err
+		}
+		// Parse
+		configBytes, err := g.FSUtil.OpenAndReadFile(filepath.Join(tempPath, entry.GetConfigFile()))
+		if err != nil {
+			return err
+		}
+		opencontrol, err := g.Parser.Parse(configBytes)
+		if err != nil {
+			return err
+		}
+		err = GetResources(tempPath, destination, opencontrol, g)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
```

Comment:
will do! (this one is on the same line as on https://github.com/opencontrol/compliance-masonry/pull/217#r75931392)